packages/docs/celo-gold-holder-guide/voting-validators.md

```diff
@@ -0,0 +1,81 @@
+
+- **Will get elected**: Locked Gold only receives voter rewards during an epoch if it is used to vote for a Validator Group that elects at least one validator during that epoch. Put another way, your vote does not contribute to securing the network or earning you rewards if your group does not receive enough other votes to elect a validator.
+
+- **Secure**: The operational security of validators is essential for everyone's use of the Celo network. All validators that participated in the Stake Off were eligible for a [security audit](https://medium.com/celoorg/the-celo-validator-community-security-audits-and-lessons-learned-e67b78cd4123). You can see scores under the "master validator challenge" column in the [Stake Off leaderboard](https://docs.google.com/spreadsheets/d/e/2PACX-1vQwk10o6YV0uriR8LuYfLqB1irjmOX_-L6Jljn3BtKlmz_R_TsUU8aI-pMqGVlu4HQKIQlQaFkUhsyl/pubhtml?gid=1970613133&single=true). Scores of 80% or greater were awarded the "Master Validator" badge, indiciating a serious proven commitment to operational security.
+
+- **Reliable**: Celo's consensus protocol relies on two-thirds of elected validators being available in order to produce blocks and process transactions. Voter rewards are directly tied to the [uptime score](../celo-codebase/protocol/proof-of-stake/validator-rewards.md#calculating-uptime-score) of all elected validators in the group for which the vote was made. Any period of consecutive downtime greater than a minute reduces a validator's uptime score.
+
+- **No recent slashing:** When validators and groups register, their Locked Gold becomes "staked", in that it is subject to penalties for conduct that could seriously adversely affect the health of the network. Voters' Locked Gold is never slashed, but voter rewards are affected by a group's [slashing penalty](../celo-codebase/protocol/proof-of-stake/validator-rewards.md#calculating-slashing-penalty), which is halved when a group or one of its validators is slashed. Look for groups with a last slashing time long in the past, ideally `0` (never), and a slashing penalty value of `1.0`.
+
+- **Runs an Attestation Service**: The [Attestation Service]() is an important service that validators can run that allows users to verify that they have access to a phone number and map it to an address. Supporting validators that run this service makes it easier for new users to begin using Celo.
```

Comment:
Did you forget to add a link to "Attestation Service"?

Reply:
I did!